test/functional/publications_controller_test.rb

```diff
@@ -339,7 +343,10 @@ def test_title
   end
 
   test 'should show the publication with unspecified publication type as Not specified' do
-    get :show, params: { id: publications(:no_publication_type) }
+    publication = Factory(:publication, title: 'Publication without type')
+    publication.publication_type = nil
+    publication.save(validate: false)
```

Comment:
you could just put this in the factory call `Factory(:publication, title: 'Publication without type', publication_type: nil)`

Reply:
I actually had to revert that after running the tests. Setting publication_type in the factory creation does not pass the validation. It is possible to define a trait in the factory to remove the validation, but as it is used only here, I don't think it would make sense.